### ISSUE TITLE
starter-kit-lisp overwrites default paredit bindings

### ISSUE THREAD

**Issue by milkypostman** [technomancy/emacs-starter-kit#141]
in order to make forward and backward slurp-sexp work in the terminal, the starter-kit is using `M-(` which is, by default, a binding used by paredit.  I normally don't' mind the added bindings that starter-kit adds but normally they aren't overwriting other bindings, although maybe I am mistaken in this statement.

**Comment by technomancy** [comment]
I agree that this sucks, but the default paredit bindings suck more. I'd be happy to change it to something less likely to conflict if we ever do a breaking version bump, but I'm not sure that will happen at this point.

**Comment by ljos** [comment]
I have been overwriting this again myself, but I recently changed to autoloading everything, and then I start getting problems with when things are happening and that the starter-kit bindings will always be the ones set. This is incredibly annoying. Especially since `paredit-backward-slurp-sexp` is even already bound to `C-(` (and I like the original behavior). I understand that this does not work in the terminal, but it would be nice if there was a way to turn this off at least.
